### DISCUSSION TITLE
Disable Default Transaction?

### DISCUSSION THREAD

**Discussion by SKFrozenCloud** [go-gorm/gorm#7069] [Q&A] [answered]
Hi,

So PostgreSQL executes every query as a transaction:

"PostgreSQL actually treats every SQL statement as being executed within a transaction. If you do not issue a BEGIN command, then each individual statement has an implicit BEGIN and (if successful) COMMIT wrapped around it. A group of statements surrounded by BEGIN and COMMIT is sometimes called a transaction block."
-https://www.postgresql.org/docs/current/tutorial-transactions.html

What does `SkipDefaultTransaction` do in PostgreSQL? Is it only relevant for other databases like SQLite?

**Comment by SKFrozenCloud** [comment, marked as answer]
So there are other functions in GORM which executes multiple queries and thus this makes the db consistent.

I think.

**Reply by CEastWind** [reply]
for example ? what functions?
